### DISCUSSION TITLE
error while creating load balancer

### DISCUSSION THREAD

**Discussion by Annepuvarunkumar** [learndevopsonline/learndevopsonline#1688] [Q&A]
![Screenshot 2023-12-21 113145](https://github.com/learndevopsonline/learndevopsonline/assets/139056909/d31a44c7-461c-46b6-8bae-97d5ab007f92)

public Ip: 44.202.54.200

**Comment by sreedharm07** [comment]
it is getting null that means the output is getting in another format you are building lookup in another formate
just check the output once  by giving output code
and according to that output observer that how that was getting then  build lookup code
